pkg/dagger.io/dagger/values.cue

```diff
@@ -0,0 +1,6 @@
+package dagger
```

Comment:
Why not in `scratch.cue`?

Reply:
See reasoning in https://github.com/dagger/dagger/issues/1997#issuecomment-1100039100